app/app/src/main/kotlin/com/hedvig/app/authenticate/BankIdLoginViewModel.kt

```diff
@@ -133,9 +142,11 @@ sealed interface BankIdLoginViewState {
   object Loading : BankIdLoginViewState {
     override fun toString(): String = "Loading"
   }
-  object Error : BankIdLoginViewState {
+
+  data class Error(val message: String) : BankIdLoginViewState {
     override fun toString(): String = "Error"
```

Comment:
Maybe edit the toString here to have the message in there too. This is used for the logs only, so more info would always be good. Something like `...: String = "Error: message:$message"`